storage/redis_cluster.go

```diff
@@ -425,66 +284,19 @@ func (r *RedisCluster) GetKeys(filter string) []string {
 // GetKeysAndValuesWithFilter will return all keys and their values with a filter
 func (r *RedisCluster) GetKeysAndValuesWithFilter(filter string) map[string]string {
 	if err := r.up(); err != nil {
-		log.Debug(err)
-		return nil
-	}
-	keys := r.GetKeys(filter)
-	if keys == nil {
-		log.Error("Error trying to get filtered client keys")
 		return nil
 	}
 
-	if len(keys) == 0 {
-		return nil
-	}
-
-	for i, v := range keys {
-		keys[i] = r.KeyPrefix + v
```

Comment:
what happens with this `r.KeyPrefix + v`?

Reply:
Re-added